data/json/npcs/items_generic.json

```diff
@@ -181,10 +181,7 @@
       ["bacon", 2],
       ["banana", 1],
       ["bandages", 25],
-      ["barrel_big", 1],
       ["barrel_ported", 1],
-      ["barrel_rifled", 1],
-      ["barrel_small", 1],
```

Comment:
Small barrel is removed from the item group, but doesn't moved to the `obsolete.json`?